### PR TITLE
Added NFC text tag code scanner section

### PR DESCRIPTION
**Pull request by SeqLaz**
Added NFC text tag code scanner description in the search bar section

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3121 #4539 Implement near-field tag/target reading capability](https://app.clickup.com/t/2192114/QF-3121)

File: documentation/how-to/search.en.md [hunk 1]
```diff
@@ -41,6 +41,14 @@ actions :
     feature will be highlighted in yellow).
 2.  Tap on the *attributes button* to open its attributes.
 
+
+In addition, the Code Reader offers the capability to read NFC text tags.
```
Comment (suricactus) [inline documentation/how-to/search.en.md:45]: I would add a more prominent section about this. Remind the use they need to enable their NFC and that it is consuming more power.

**Review by suricactus — COMMENTED** [review]
Does the NFC work only for searching, or also for filling a textual field?

**Comment by nirvn** [comment]
@suricactus , answer to your last question: both.

File: documentation/how-to/search.en.md (hunk 1 above)
Comment (SeqLaz) [reply documentation/how-to/search.en.md:45]: Great!

**Comment by suricactus** [comment]
Can you please add the info also here https://docs.qfield.org/how-to/digitize/?h=qr+code#attribute-form

File: documentation/how-to/digitize.en.md [hunk 2]
```diff
@@ -73,6 +73,13 @@ Once the codes have been successfully scanned, users can validate their choice b
 
 !![](../assets/images/digitizing-with-qr-code-4-it-worked.png)
 
+In addition, the Code Reader offers the capability to read NFC text tags.
+
+!![](../assets/images/code-reader-nfc-text-tag.png)
+
+!!! note
+    While both the QR code camera and the NFC text tag detector are enabled by default when you open the Code Reader. You have the flexibility to disable either of these features to ensure that your device's battery is not used unnecessarily to activate hardware that you may not need at the moment.
```
Comment (m-kuhn) [inline documentation/how-to/digitize.en.md:81]:
```suggestion
    Both the QR code camera and the NFC text tag detector are enabled by default when you open the Code Reader. You have the flexibility to disable either of these features to ensure that your device's battery is not used unnecessarily to activate hardware that you may not need at the moment.
```

Either we remove the "While" or we remove the period `.` in "Code Reader. You [...]"

File: documentation/how-to/search.en.md [hunk 3]
```diff
@@ -20,17 +20,17 @@ The value entered in the search bar will be used to find features with
 matching attribute values. A minimum of three characters is required to
 start the search.
 
-!![image](../assets/images/search-bar.png)
+!![](../assets/images/search-bar.png)
 
 Additionally, you have the option to employ the code scan functionality for this purpose.
```
Comment (suricactus) [inline documentation/how-to/search.en.md:25]:
```suggestion
## Search with code scanner

Additionally, you have the option to employ the code scan functionality for this purpose.
```

File: documentation/how-to/search.en.md (hunk 1 above)
Comment (suricactus) [inline documentation/how-to/search.en.md:45]:
```suggestion
### Search with NFC

In addition, the Code Reader offers the capability to read NFC text tags.
```

**Review by suricactus — APPROVED** [review]

**Pull request merged**